### PR TITLE
Issue #25846 Updating the API docs of the tf.math.add function

### PR DESCRIPTION
**Pull request by HarshSulakhe**
#25846

### CODE REVIEW

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
You'll have to remove unrelated changes

**Comment by gbaned** [comment]
@HarshSulakhe Could you please check reviewer comments and keep us posted. Thanks!

**Comment by HarshSulakhe** [comment]
Sorry for the delay and a large number of commits, I'm a little new to Git, messed up a bit.
Have made the changes that were requested.

File: tensorflow/core/api_def/base_api/api_def_Acos.pbtxt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 op {
   graph_op_name: "Acos"
   summary: "Computes acos of x element-wise."
-}
+  }
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_Acos.pbtxt]: Extra space should not be added here

File: tensorflow/core/api_def/base_api/api_def_Add.pbtxt [hunk 2]
```diff
@@ -2,6 +2,13 @@ op {
   graph_op_name: "Add"
   summary: "Returns x + y element-wise."
   description: <<END
+  Given two input tensors, tf.math.add computes and returns the element wise sum of the tensors.
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_Add.pbtxt]: This line is too long. pylint would complain about it

File: tensorflow/core/api_def/base_api/api_def_Add.pbtxt [hunk 3]
```diff
@@ -2,6 +2,13 @@ op {
...
+  Input range is `[-inf, inf]` and output range is `[-inf, inf]`.
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_Add.pbtxt]: I don't think we should include input range if all inputs are supported. Similarly, we should not include the output range if any output can be provided.

Also, the intervals are wrong: from a mathematical point of view, the intervals should be open (`(-inf, inf)`) as the infinities cannot be reached. However, from a programming point of view not all values can be represented so maybe it's `[-float_max, float_max]` or something. Better to just not include the line at all.

File: tensorflow/core/api_def/base_api/api_def_Add.pbtxt [hunk 4]
```diff
@@ -2,6 +2,13 @@ op {
...
+  ```python
+  x = tf.constant([1.0, -0.5, 3.4, 0.2, 0.0, -2] , dtype = tf.float32)
+  y = tf.constant([1.7, -2.3, 11.4, -1.0, 0.0, -5] , dtype = tf.float32)
+  ```
+  tf.math.add(x,y) ==> [2.7  -2.8  14.8  -0.8  0.0  -7]
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_Add.pbtxt]: Shouldn't this line be in the triple quoted blocks too?

Also, if you want to show a proper example, a better comment would be in the doctest format:

```python
>>> x = tf.constant([1.0, -0.5, 3.4, 0.2, 0.0, -2] , dtype = tf.float32)
>>> y = tf.constant([1.7, -2.3, 11.4, -1.0, 0.0, -5] , dtype = tf.float32)
>>> tf.math.add(x,y)
[2.7  -2.8  14.8  -0.8  0.0  -7]
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/core/api_def/base_api/api_def_Add.pbtxt (hunk 4 above)
Comment (HarshSulakhe) [reply tensorflow/core/api_def/base_api/api_def_Add.pbtxt]: Apologies for the errors will rectify them at the earliest.

File: tensorflow/core/api_def/base_api/api_def_Acos.pbtxt [hunk 5]
```diff
@@ -2,3 +2,4 @@ op {
   graph_op_name: "Acos"
   summary: "Computes acos of x element-wise."
 }
+
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_Acos.pbtxt]: This extra line should not be here, this file should not be touched at all.

I suggest `git checkout HEAD -- tensorflow/core/api_def/base_api/api_def_Acos.pbtxt`

File: tensorflow/core/api_def/base_api/api_def_Add.pbtxt [hunk 6]
```diff
@@ -2,6 +2,13 @@ op {
   graph_op_name: "Add"
   summary: "Returns x + y element-wise."
   description: <<END
+  Given two tensors, tf.math.add returns the element wise sum of the tensors.
+  ```python
+  >>>x = tf.constant([1.0, -0.5, 3.4, 0.2, 0.0, -2] , dtype = tf.float32)
+  >>>y = tf.constant([1.7, -2.3, 11.4, -1.0, 0.0, -5] , dtype = tf.float32)
+  >>>tf.math.add(x,y) ==> [2.7  -2.8  14.8  -0.8  0.0  -7]
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_Add.pbtxt]: Put the result value on a different line and remove the triple backticks around the block, please.

File: tensorflow/python/keras/activations.py [hunk 7]
```diff
@@ -179,21 +179,28 @@ def softsign(x):
 def relu(x, alpha=0., max_value=None, threshold=0):
   """Rectified Linear Unit.
 
-  With default values, it returns element-wise `max(x, 0)`.
-
-  Otherwise, it follows:
+  The Rectified Linear Unit function is:
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: This file should not be in this PR. See the `git checkout` command above to revert.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Note that adding multiple files, will make this harder to import, even if you end up reverting them at the end.

The recommended workflow is

```console
git checkout -b branch-name-describing-feature-you-want-to-do
# do some work
git commit ...
git push
# make the PR
git checkout master
# once the PR is accepted you can delete the branch
```

This way, unrelated changes are all in their own branch and don't interfere.

**Comment by gbaned** [comment]
@HarshSulakhe Could you please check reviewer comments and keep us posted. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by mihaimaruseac** [comment]
Closing as stale.

**Pull request closed without merging**